zeppelin-zengine/src/main/java/org/apache/zeppelin/helium/HeliumBundleFactory.java

```diff
@@ -194,7 +194,12 @@ public boolean accept(File pathname) {
 
     try {
       out.reset();
-      npmCommand("install --loglevel=error");
+      String npmCommand =
```

Comment:
MINOR: Variable name looks duplicated with the function name `npmCommand`. It would be nice if we use other name.